### PR TITLE
Fix concept label change to be propagated to element labels

### LINKED ISSUE #97: ng-change updates are not persisted
on the overview page changing the label might get lost
reported by Matthias supercell-beta bug 3.mp4

### PR DESCRIPTION
**Pull request by dknochen**
fixes #97

### CODE REVIEW

**Comment by dknochen** [comment]
@wcandillon could you please review? 

I also added a helper script to start the element explorer.

File: Gruntfile.js [hunk 1]
```diff
@@ -167,11 +167,6 @@ module.exports = function (grunt) {
                 configFile: './karma.conf.js'
             },
             dev: {
-                browsers: ['Chrome'],
```
Comment (wcandillon) [inline Gruntfile.js:170]: This was dead grunt code

File: app/report/taxonomy/concept/overview/overview-test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+describe('Check Concept updating ', function () {
```
Comment (wcandillon) [inline app/report/taxonomy/concept/overview/overview-test.js:1]: We limited the unit test to the assertions that are isolated in the overview controller.

File: tests/e2e/basic-scenario.js [hunk 3]
```diff
@@ -46,6 +46,14 @@ describe('Report', function(){
         report.taxonomy.getConcept(conceptName).createElement();
         expect(report.taxonomy.elements.count()).toBe(1);
     });
+    
+    it('Renames the concept label', function(){
```
Comment (wcandillon) [inline tests/e2e/basic-scenario.js:50]: We added the e2e test case for this bug.

File: app/report/report.js [hunk 4]
```diff
@@ -4,37 +4,51 @@ angular.module('report-editor')
 .controller('ReportCtrl', function($q, $rootScope, $scope, Session, API, Report, report){
     $scope.report = report;
 
-    var lastKnowSavedModel = $scope.report.model;
-    var lastKnowSavedModelAsString = JSON.stringify(lastKnowSavedModel);
-    var saveCanceler;
+    var lastSavedModel;
```
Comment (wcandillon) [inline app/report/report.js:7]: The testing of this bug outlined a bug is on strategy to automatically save reports.

**Comment by wcandillon** [comment]
@dknochen We updated this branch I hope it make sense. We removed the following file: https://www.dropbox.com/s/dsddfwh3z8cnzvs/explore.sh it should be integrated in the gruntfile. I think it should be too much work to do it.

**Comment by dknochen** [comment]
@wcandillon Added an issue for it: https://github.com/28msec/nolap-report-editor/issues/118

**Pull request merged**